src/main/java/org/jitsi/jicofo/bridge/BridgeSelectionStrategy.java

```diff
@@ -63,9 +62,9 @@
     private int totalSplitDueToLoad;
 
     /**
-     * Maximum participants per bridge in one conference.
+     * Maximum participants per bridge in one conference, or {@code -1} for no maximum.
      */
-    private int maxParticipantsPerBridge = Integer.MAX_VALUE;
+    private int maxParticipantsPerBridge = BridgeConfig.config.maxBridgeParticipants();
```

Comment:
Any pattern to where you  do `BridgeConfig.config...` vs statically importing `BridgeConfig.config` and just do `config...` ?